static/src/javascripts/projects/commercial/modules/messenger/type.ts

```diff
@@ -0,0 +1,20 @@
+import { isString } from '@guardian/libs';
+import fastdom from '../../../../lib/fastdom-promise';
+import type { RegisterListener } from '../messenger';
+
+const setType = (type: string, adSlot: Element) =>
```

Comment:
I'm not sure what `type` is describing here? Is it the type of advert? Would `adType` be a bit more descriptive?

Reply:
Agreed, `adSlotType`?